packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -1778,6 +1793,11 @@ class TwoDimensionalScrollable extends StatefulWidget {
   /// This value applies to both axes.
   final bool excludeFromSemantics;
 
+  /// {@macro flutter.widgets.scrollable.hitTestBehavior}
+  ///
+  /// This value applies to both axes.
```

Comment:
There should probably be a test for this like the second test you added. That one only tests the 1D scrollable. Much of the 2D classes are abstract, but there are concrete classes for testing in flutter/packages/flutter/test/widgets/two_dimensional_utils.dart.